lib/beaker/host/pswindows/pkg.rb

```diff
@@ -7,15 +7,21 @@ def check_for_command(name)
   end
 
   def check_for_package(name)
-    raise "Cannot check for package #{name} on #{self}"
+    #HACK NOOP
+    #raise "Cannot check for package #{name} on #{self}"
+    0
   end
 
   def install_package(name, cmdline_args = '')
```

Comment:
:+1: for adding this in. I can understand why this is a no-op at the moment, but this will give me (us) something to think about and extend for the different packing types available.